server/src/main/java/io/druid/server/coordinator/LoadQueuePeon.java

```diff
@@ -326,6 +311,29 @@ public void run()
     }
   }
 
+  public void start()
+  {
+    ScheduledExecutors.scheduleAtFixedRate(
+        processingExecutor,
+        config.getLoadQueuePeonStartDelay(),
+        Period.millis(50).toStandardDuration(),
```

Comment:
Errr, I had expected you to make the repeat duration configurable, not the start delay.  We often also reuse the repeat duration for the start delay, which I think we could do here too.